homeassistant/components/season/sensor.py

```diff
@@ -53,28 +57,30 @@
 )
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+def setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
+) -> None:
     """Display the current season."""
-    if None in (hass.config.latitude, hass.config.longitude):
-        _LOGGER.error("Latitude or longitude not set in Home Assistant config")
-        return False
+    if (latitude := hass.config.latitude) is None:
```

Comment:
This changes logic? As in, it only checks one now..

Reply:
What is the point of checking the longitude? It is not used that I could see.

Reply:
The season tests are failing anyway - so I will move this out.